src/app/shared/modules/selectors/faction-selector/quest-faction-selector-btn.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { BsModalService } from 'ngx-bootstrap/modal';
+
+import { BaseSelectorBtnComponent } from '../base-selector/base-selector-btn.component';
+import { QuestFactionSelectorModalComponent } from './quest-faction-selector-modal.component copy';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
we should try making new components with `standalone: true` and ideally migrate all the old ones to standalone too later on

Reply:
build and tests fails, let's do this in another occasion/PR